content/sensu-go/5.18/getting-started/enterprise.md

```diff
@@ -11,25 +11,33 @@ menu:
-[Contact the Sensu sales team][1] for a personalized demo and free trial.
-All commercial features are available for free in the packaged Sensu Go distribution up to an entity limit of 100.
-See the [announcement on our blog][7] for more information.
+All commercial features are available for free in the packaged Sensu Go distribution [up to an entity limit of 100][7].
+
+If you have more than 100 entities, [contact the Sensu sales team][1] for a personalized demo and free trial.
 
 ## Commercial features in Sensu Go
 
+- **Protect your sensitive information** with [secrets management][19].
+Authenticate with Sensu's built-in secrets provider or HashiCorp Vault and avoid exposing usernames, passwords, and access keys in your Sensu configuration.
+- **Manage your monitoring resources across multiple data centers, cloud regions, or providers** and mirror changes to follower clusters with [federation][20].
+Federation affords visibility into the health of your infrastructure and services across multiple distinct Sensu instances within a single web UI.
+- **Use mutual transport layer security (mTLS) authentication** to [secure your Sensu agents and backends][21].
 - **Manage resources from your browser**: Create, edit, and delete checks, handlers, mutators, and filters using the Sensu [web UI][8], and access the Sensu web UI homepage.
-- **Authentication providers**: Scale Sensu role-based access control (RBAC) with [LDAP and Active Directory integrations][9].
-- **Scalable resource filtering** designed for large installations: Use label and field selectors to filter [Sensu API][4] responses, [sensuctl][5] outputs, and Sensu [web UI][6] views using custom labels and a wider range of resource attributes.
-- **Event logging**: Log event data to a file you can use as an input source for your favorite data lake solution.
-The [event logging][10] functionality provides better performance and reliability than event handlers.
-- **Enterprise-tier assets**: Connect your monitoring event pipelines to industry-standard tools like ServiceNow and Jira with [enterprise-tier assets][11].
-- **Enterprise-scale event storage**: Scale your Sensu instance and handle high volumes of events with a [PostgreSQL event store][12].
-- **Enterprise-class support**: Rest assured that with [Sensu support][13], help is available if you need it.
+- **Scale Sensu role-based access control (RBAC)** with [Lightweight Directory Access Protocol (LDAP) and Active Directory (AD) integrations][9].
+- **Use powerful filter resources** designed for large installations. With label and field selectors, you can filter [Sensu API][4] responses, [sensuctl][5] outputs, and Sensu [web UI][6] views using custom labels and a wider range of resource attributes.
+- **Log event data** to a file you can use as an input source for your favorite data lake solution.
+Sensu's [event logging][10] provides better performance and reliability than event handlers.
```

Comment:
```suggestion
```

I'm not sure how this is true. We could strike it and move the link into the surviving sentence above.

Reply:
👍